docs/source/index.rst

```diff
@@ -55,3 +55,11 @@ External resources
    sdk/hotpocket/index
    sdk/evernode/index
    sdk/patterns/index
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   :caption: API
+
+   Evernode JS Client <APIs/evernode-js-api>
```

Comment:
Add a simple description. Probably we can link this in `libraries.md`

evernode-js-api - npm library link. Mention that it consists of wrapped evernode related operations in xrpl ledger and helpers
everpocket-api - npm library link. Mention that it consists of contract development tools and helpers and xrpl, and evernode integrations for contracts